vips/image.go

```diff
@@ -414,9 +424,25 @@ func (r *ImageRef) RemoveICCProfile() error {
 	return nil
 }
 
+// deprecated: use optimize
 func (r *ImageRef) TransformICCProfile(isCmyk int) error {
-	out, err := vipsTransformICCProfile(r.image, isCmyk)
+	out, err := vipsOptimizeICCProfile(r.image, isCmyk)
+	if err != nil {
+		return err
+	}
+	r.setImage(out)
+	return nil
+}
+
+func (r *ImageRef) OptimizeICCProfile() error {
```

Comment:
why do we need both OptimizeICCProfile and TransformICCProfile ?

Reply:
backwards compatibility until I kill the old function